### PR TITLE
Portfolio content

### PR DESCRIPTION
**Pull request by lucyholubova**

### CODE REVIEW

File: css/portfolio.css [hunk 1]
```diff
@@ -0,0 +1,205 @@
+* {
+	box-sizing: border-box;
+/*	border: 1px solid red;*/
```
Comment (snack-attack) [inline css/portfolio.css]: delete commented code if you're not using it

File: css/portfolio.css [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    position: absolute;
+    top: 0px;
+    left: 0;
+    min-height: 100vh;
+    min-width: 100%;
+    /*transform: translateX(-50%);*/
+    /*left: 50%;*/
+}
+
+.header{
```
Comment (snack-attack) [inline css/portfolio.css]: add space after class for consistency

File: css/portfolio.css [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    font-size: small;
+    position: relative;
+    z-index: 10;
+    height: 100vh;
+    overflow: visible;
+    /*background-color:#333;*/
+}
+
+.header-title{
```
Comment (snack-attack) [inline css/portfolio.css]: add space after class name

File: css/portfolio.css [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    left: 50vw;
+    transform: translateX(-50%);
+}
+
+section h2.contacts {
+    margin-top: -18px;
+}
+
+.mybio {
+    font-size: large;
```
Comment (snack-attack) [inline css/portfolio.css]: add a media query to this class to make the width 100% on mobile

Also consider setting the font-size to a specific rem size, then you can control the size on mobile, desktop, etc

File: css/portfolio.css [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+    border-left-width: 10px;
+    height: 90px;
+    position: absolute;
+    top: 10px;
+    left:50%;
+    z-index: 10;
+}
+
+
+.vl {
```
Comment (snack-attack) [inline css/portfolio.css:68]: this div is not working on the bio section on mobile

File: css/portfolio.css [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+    border-left-width: 10px;
+    border-left-style: solid;
+    border-left-color: green;
+    height: 90px;
+    position: absolute;
```
Comment (snack-attack) [inline css/portfolio.css:73]: fine to leave this absolute on mobile, but you need to add a media query to move it around in a smaller view so it doesn't cover the text

File: css/portfolio.css [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+    height: 100vh;
+    background-color: whitesmoke;
+
+
+}
+
+.details{
+
+
+    display: inline-block;
```
Comment (snack-attack) [inline css/portfolio.css:155]: add a media query for making this display block on mobile

File: css/portfolio.css [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+    right: 0;
+    /*background: blue;*/
+    z-index: 5;
+}
+
+audio{
+    display:none;
+}
+
+p {
```
Comment (snack-attack) [inline css/portfolio.css:183]: you'll probably need a media query for your <p> elements to fix spacing issues

**Comment by snack-attack** [comment]
good job!

**Review by snack-attack — CHANGES_REQUESTED** [review]

File: css/portfolio.css (hunk 4 above)
Comment (lucyholubova) [reply css/portfolio.css]: thank youuuuu

File: css/portfolio.css [hunk 9]
```diff
@@ -0,0 +1,195 @@
+* {
+	box-sizing: border-box;
```
Comment (mporam) [inline css/portfolio.css]: fix indentation

File: css/portfolio.css [hunk 10]
```diff
@@ -0,0 +1,195 @@
...
+    font-family: Modenine;
+    src: url('../font/MODENINE.TTF');
+    src: url('../font/MODENINE.woff');
+}
+
+body {
+    font-family: Modenine;
+    margin: 0;
+}
+
```
Comment (mporam) [inline css/portfolio.css:15]: extra line breaks, just 1 between each selector

File: css/portfolio.css [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+    right: 0;
+    left: 0;
+    overflow: hidden;
+    z-index: 2;
+    height: 100vh;
+}
+
+.fullscreen-bg__video {
+    position: absolute;
+    top: 0px;
```
Comment (mporam) [inline css/portfolio.css]: for 0 values you dont need a unit, so just `top: 0;` will do

File: css/portfolio.css [hunk 12]
```diff
@@ -0,0 +1,195 @@
...
+.header {
+    font-size: small;
+    position: relative;
+    z-index: 10;
+    height: 100vh;
+    overflow: visible;
+}
+
+.header-title {
+    color: whitesmoke;
```
Comment (mporam) [inline css/portfolio.css]: use a hex value please

File: css/portfolio.css [hunk 13]
```diff
@@ -0,0 +1,195 @@
...
+    width: 100%;
+    padding: 0 15px;
+}
+
+.logo {
+    border-left-width: 10px;
+    height: 90px;
+    position: absolute;
+    top: 10px;
+    left:50%;
```
Comment (mporam) [inline css/portfolio.css]: use this format:
`property: value;`
not:
`property:value;`

File: css/portfolio.css [hunk 14]
```diff
@@ -0,0 +1,195 @@
...
+
+}
+
+.portfolio {
+    position: relative;
+    overflow: hidden;
+    width: 100%;
+    height: 100vh;
+    background-color: whitesmoke;
+
```
Comment (mporam) [inline css/portfolio.css:129]: remove line break

File: css/portfolio.css [hunk 15]
```diff
@@ -0,0 +1,195 @@
...
+.mybio {
+    max-width: 550px;
+    padding: 0 15px;
+    position: absolute;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    left: 50%;
+    width: 100%;
+
```
Comment (mporam) [inline css/portfolio.css:120]: remove line break

File: css/portfolio.css [hunk 16]
```diff
@@ -0,0 +1,195 @@
...
+    position: relative;
+    overflow: hidden;
+    width: 100%;
+    height: 100vh;
+    background-color: whitesmoke;
+
+}
+
+.numbers {
+    color:red;
```
Comment (mporam) [inline css/portfolio.css]: use hex color

File: css/portfolio.css [hunk 17]
```diff
@@ -0,0 +1,195 @@
...
+    margin:0 auto;
```
Comment (mporam) [inline css/portfolio.css]: property:SPACE value;

File: css/portfolio.css [hunk 18]
```diff
@@ -0,0 +1,195 @@
...
+    width: 100px;
+    font-size: 40px;
+    padding-right: 210px;
+    margin-top: -20px;
+}
+
+.numbers a {
+    text-decoration: none;
+    color:red;
```
Comment (mporam) [inline css/portfolio.css]: hex value

File: css/portfolio.css [hunk 19]
```diff
@@ -0,0 +1,195 @@
...
+}
+
+.contact_main {
+    position: relative;
+    overflow: hidden;
+    width: 100%;
+    height: 100vh;
+    background-color: whitesmoke;
+
```
Comment (mporam) [inline css/portfolio.css]: remove line break

File: css/portfolio.css [hunk 20]
```diff
@@ -0,0 +1,195 @@
...
+}
+
+.details {
+
```
Comment (mporam) [inline css/portfolio.css]: remove line break

File: css/portfolio.css (hunk 1 above)
Comment (lucyholubova) [reply css/portfolio.css]: deleted it

File: css/portfolio.css (hunk 2 above)
Comment (lucyholubova) [reply css/portfolio.css]: done

File: css/portfolio.css (hunk 3 above)
Comment (lucyholubova) [reply css/portfolio.css]: done

File: css/portfolio.css (hunk 5 above)
Comment (lucyholubova) [reply css/portfolio.css:68]: i just changed the margin a moved it more to the right

File: css/portfolio.css (hunk 6 above)
Comment (lucyholubova) [reply css/portfolio.css:73]: I moved the text instead

File: portfolio.html [hunk 21]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+
```
Comment (mporam) [inline portfolio.html:2]: remove line break

File: portfolio.html [hunk 22]
```diff
@@ -0,0 +1,90 @@
...
+<html lang="en">
+
```
Comment (mporam) [inline portfolio.html:4]: remove line break

File: portfolio.html [hunk 23]
```diff
@@ -0,0 +1,90 @@
...
+<head>
+
```
Comment (mporam) [inline portfolio.html:6]: remove line break

File: portfolio.html [hunk 24]
```diff
@@ -0,0 +1,90 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <link href="css/normalize.css" rel="stylesheet" type="text/css">
+    <link href="css/portfolio.css" rel="stylesheet" type="text/css">
+    <title>Portfolio</title>
+</head>
+
```
Comment (mporam) [inline portfolio.html:12]: remove line break

File: portfolio.html [hunk 25]
```diff
@@ -0,0 +1,90 @@
...
+<body>
+<iframe src="music/s.mp3" allow="autoplay" id="audio" style="display:none"></iframe>
+<audio autoplay loop>
+	<source src="music/song.mp3" type="audio/mpeg">
+</audio>
+	<div class="fullscreen-bg">
+
```
Comment (mporam) [inline portfolio.html]: remove line break

File: portfolio.html [hunk 26]
```diff
@@ -0,0 +1,90 @@
...
+			<video class="fullscreen-bg__video" loop muted autoplay poster="img/lucy.jpg" >
```
Comment (mporam) [inline portfolio.html]: fix indentation

File: portfolio.html [hunk 27]
```diff
@@ -0,0 +1,90 @@
...
+
+
+	<section id="portfolio">
+		<a href="#contact" class="contact-link"></a>
+		<div class="portfolio">
+			<img src="img/L90.png" class="logo" alt="logo">
+			<h2 class="work">
+				<a href="#contact">WORK</a>
+				<div class="numbers">
+					<a href=# class="options1"> 1 </a>
```
Comment (mporam) [inline portfolio.html]: missing "quotes" from the #s

File: portfolio.html [hunk 28]
```diff
@@ -0,0 +1,90 @@
...
+				<a href="#header">LUCY HOLUBOVA</a>
+				<div class="details">
+					<a href="https://github.com/lucyholubv" class="contact">GITHUB</a>
+					<a href="mailto:lucy.holubova90@gmail.com" class="email">EMAIL</a>
+					<a href="https://www.instagram.com/lucccul/" class="instagram">INSTAGRAM</a>
+				</div>
+			</h2>
+			<div class="vl"></div>
+			<p> MADE WITH LOVE </p>
+
```
Comment (mporam) [inline portfolio.html]: remove line break

File: portfolio.html [hunk 29]
```diff
@@ -0,0 +1,90 @@
...
+		</div>
+	</section>
+
```
Comment (mporam) [inline portfolio.html]: remove all this white space

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by lucyholubova — COMMENTED** [review]
i eddited this

File: css/portfolio.css [hunk 30]
```diff
@@ -0,0 +1,203 @@
...
+    width: 100px;
+    font-size: 40px;
+    padding-right: 210px;
+    margin-top: -20px;
+}
+
+.numbers a {
+    text-decoration: none;
+    color: #ff0000;
+}
```
Comment (sjr-mayden) [inline css/portfolio.css:144]: This needs to be deleted

File: css/portfolio.css (hunk 19 above)
Comment (sjr-mayden) [reply css/portfolio.css]: Still needs removing

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

**Pull request merged**